string new_string
---
string change_status